### PR TITLE
feat(JOML): Fixes invisible spawning when scale is used

### PR DESCRIPTION
**Pull request by casals**
### Contains

Quick fix to #4197

### How to test

Spawn a creature with `SkeletalMeshComponent:scale` specified on its prefab.

### Outstanding before merging

- [x] This is a quick fix. We need to change all the methods in `OpenGLSkeletalMesh` using `Vector3f`, which also involves dealing with `SkeletalMeshData`.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -12,5 +12,8 @@
     </writeAnnotations>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="FrameworkDetectionExcludesConfiguration">
```
Comment (DarkWeird) [inline .idea/misc.xml:15]: It is really needs?

Comment (casals) [reply .idea/misc.xml:15]: This file is not on gitignore and it's been updated in the repo, so I really didn't bother removing it.

**Review by pollend — APPROVED** [review]

**Review by jdrueckert — CHANGES_REQUESTED** [review]
@casals I tried it with the following changes to the current `WildAnimals:develop` `babyDeer.prefab` and unfortunately the baby deer is still invisible :thinking: 

```
diff --git a/assets/prefabs/animals/babyDeer.prefab b/assets/prefabs/animals/babyDeer.prefab
index 618711d..619d1bc 100644
--- a/assets/prefabs/animals/babyDeer.prefab
+++ b/assets/prefabs/animals/babyDeer.prefab
@@ -1,10 +1,15 @@
 {
   "skeletalMesh": {
     "mesh": "deer",
-    "heightOffset": 1.0,
+    "heightOffset": -1.6,
     "material": "deerSkin",
     "animation": "deer#idle",
-    "loop": true
+    "loop": true,
+    "scale": [
+      0.5,
+      0.5,
+      0.5
+    ]
   },
   "Behavior": {
     "tree": "Behaviors:stray"
```

**Review by jdrueckert — APPROVED** [review]
Works :+1:

**Pull request merged**